form-editor-cae/src/main/java/com/tallence/formeditor/cae/handler/FormController.java

```diff
@@ -97,6 +94,13 @@ public FormProcessingResult socialFormAction(@PathVariable CMChannel currentCont
                                                HttpServletRequest request,
                                                HttpServletResponse response) throws IOException {
 
+    if (target == null || currentContext == null) {
+      // Log the form data for debugging purpose, wrapped in a LinkedHashMap to make sure, the toString method is overwritten.
+      LOG.warn("No form or context document found, cannot handle the request. Form data: {}", new LinkedHashMap<>(postData));
```

Comment:
I wonder if we should add some kind of mechanism to prevent form data from ever being logged (for compliance reasons, if a users needs/wants that).

Reply:
Discussion moved to #63. Closing this..